src/recommendations/events.md

```diff
@@ -7,16 +7,16 @@ When you deploy the `product-recommendations` module, the module tracks the foll
 
 Event |Description
 --- | ---
-`add-to-cart` | Triggered when the shopper adds an item to their cart
-`add-to-wishlist` | Triggered when the shopper adds an item to their wishlist
-`apply-discount-action` | Triggered when the shopper applies a discount to their cart
-`edit-product-qty` | Triggered when the shopper modifies the number of items in their cart
+`add-to-cart` | Triggered when a cart updates with a new item
+`add-to-wishlist` | Triggered when a wishlist updates with a new item
+`apply-discount-action` | Triggered when a discount is applied to a cart
+`edit-product-qty` | Triggered when the number of items in a cart is modified
 `initiate-checkout` | Triggered when the shopper clicks the **Proceed to Checkout** button
-`instant-purchase` | Triggered when the shopper selects the **Add to Cart** button on a category page
-`place-order` | Triggered when the shopper clicks the **Place Order** button
-`remove-discount-action` | Triggered when the shopper removes a discount from their cart
-`remove-from-cart` | Triggered when the shopper removes an item from their cart
-`remove-from-wishlist` | Triggered when the shopper removes an item from their wishlist
+`instant-purchase` | Triggered when the shopper completes an instant purchase. Typically triggered on a product page, but could be triggered on product listing pages like category and search results
+`place-order` | Triggered when the customer's order is complete
```

Comment:
is *submitted*

"completed" could be conflated with the "complete" order status once it is invoiced, paid, and shipped